### DISCUSSION TITLE
Question related to 'Schedule Calculate'

### DISCUSSION THREAD

**Discussion by UmeshGitHubUser** [SFDO-Community/declarative-lookup-rollup-summaries#1080] [Q&A]
Hi All,

I am currently working on DLRS, I have set up DLRS on Account, Opportunities.
On Opps, I am rolling up the quantity of the Products used.
On Account, I am rolling the Products.

When ever the user is updating the records, I see that Rollup being updated, but for old accounts which have Opportunities are not being updated. So I was planning to use "Schedule Calculate" feature, which works on field by field. I mean the size of jobs will increase and may affect the performance as I do have many other jobs. So let me know what is the best way to use this feature considering above scenario. I need help and suggestion on how should I proceed?

**Comment by aheber** [comment]
@UmeshGitHubUser thanks for asking a great question! Normally we handle all "how do I use it" questions in the Trailhead group, https://trailhead.salesforce.com/trailblazer-community/groups/0F9300000009O5pCAE. If you have followup questions then I'll direct you there.

I'll do my best to answer this one, hopefully I'm understanding what you're asking.

I think that you're saying you've added a DLRS rollup job and anytime the child records are updated it is correctly updating the parent. Now you're looking at parents that haven't been updated and wondering how you "reconcile" them, or populate the correct value in the database.

To do that you'll typically use the "Full Calculate" action on the Rollup Summary screen. If you want to limit the scope of that run to specific records then you can enter criteria, in the form of a WHERE clause, on that screen and kick it off. That will reconcile all of the records once, ideally DLRS will keep everything in sync going forward. If for some reason you need to perform this action more often then you use the "Schedule Full Calculate," usually I find I need that if I use time criteria in the rollup definition.

Hope that helps, again if you need more help I'll send you out to the Trailhead group linked above.

**Reply by UmeshGitHubUser** [reply]
@aheber Thanks for that, I will ask the question in trailhead community,
